sdk/src/client/api/address.rs

```diff
@@ -117,7 +150,29 @@ impl SecretManager {
         )
     }
 
-    /// Get a vector of EVM address strings
+    /// Generates a single EVM address hex string.
+    pub async fn generate_evm_address(
+        &self,
+        coin_type: u32,
+        account_index: u32,
+        address_index: u32,
+        options: impl Into<Option<GenerateAddressOptions>> + Send,
+    ) -> Result<String, ClientError> {
+        Ok(prefix_hex::encode(
+            SecretManage::generate_evm_addresses(
+                self,
+                coin_type,
+                account_index,
+                address_index..address_index + 1,
+                options,
+            )
+            // Panic: if the secret manager hasn't failed then there must be an address.
+            .await?[0]
+                .as_ref(),
```

Comment:
are you okay, rustfmt?

Reply:
sometimes it just stops working for a file if something is wrong :/